File: core/src/main/scala/pl/edu/agh/multiscalemodelling/processsimulation/SimulationCell.java
package pl.edu.agh.multiscalemodelling.processsimulation;

import com.badlogic.gdx.graphics.Color;
import pl.edu.agh.multiscalemodelling.engine.logic.Cell;

import java.util.HashMap;

public class SimulationCell extends Cell {

    protected static HashMap<Integer, Color> seedList;
    protected Integer seedID;
    protected Integer nextSeedID;

    public SimulationCell(int x, int y) {
        super(x, y);
    }

}

File: core/src/main/scala/pl/edu/agh/multiscalemodelling/processsimulation/SimulationCell.scala
package pl.edu.agh.multiscalemodelling.processsimulation

import java.util

import com.badlogic.gdx.graphics.Color
import pl.edu.agh.multiscalemodelling.engine.logic.Cell

object SimulationCell {
  protected var seedList: util.HashMap[Integer, Color] = null
}

class SimulationCell(x: Int, y: Int) extends Cell(x, y) {
/*  var nextSeedID: Integer = null*/
}